DeUrgenta/Utils/Color+Extensions.swift

```diff
@@ -22,6 +22,14 @@ extension Color {
         Color("FieldBorder")
     }
 
+    static var blue: Color {
```

Comment:
I think we already have a blue color defined by the SDK. Can we find a new name, just to avoid confusion?

Reply:
ok